apps/studio/tests/common.ts

```diff
@@ -0,0 +1,190 @@
+import { dirname, join } from "path"
+import { fileURLToPath } from "url"
+import type { StartedNetwork, StartedTestContainer } from "testcontainers"
+import { GenericContainer, Wait } from "testcontainers"
+import { z } from "zod"
+
+type ContainerType = "database" | "studio"
+export const CONTAINER_CONFIGURATIONS: {
```

Comment:
question(non-blocking): sorry a bit dumb, how come we don't need to do the same for the mockpass container here?

Reply:
we don't need this, lemme remove it. we do `setup/teardown` via `docker compose` for e2e, and we only setup postgres for integration